src/index.ts

```diff
@@ -1 +1,15 @@
 export * from "./coinbase/coinbase";
+export * from "./coinbase/wallet";
```

Comment:
why do we not catch this sooner in our tests? should we use an e2e test that uses a released npm package instead of locally built sdk?

Reply:
The classes return a type in TypeScript. We avoid multiple exports for other classes to ensure interactions are limited to the main Coinbase class. If users want to import a specific class, they can do so through direct imports as described in our documentation.

Reply:
Yes we should use released packages for e2e tests. Several versions.

IMO the main way to catch things like this though is to have the team building with the SDK on a regular basis!